### PR TITLE
Fixed custom templates for static builds

### PR DESCRIPTION
**Pull request by tboerger**
Now the custom templates are finally back! :bomb:

### CODE REVIEW

File: modules/auth/oauth2/oauth2.go [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
 import (
-	"code.gitea.io/gitea/modules/setting"
 	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
 	"github.com/gorilla/sessions"
 	"github.com/markbates/goth"
 	"github.com/markbates/goth/gothic"
+	"github.com/markbates/goth/providers/github"
+	"github.com/satori/go.uuid"
 	"net/http"
```
Comment (appleboy) [inline modules/auth/oauth2/oauth2.go]: coding style about import sorter.

Comment (tboerger) [reply modules/auth/oauth2/oauth2.go]: That's not my code, this change is anyway only there because it have not been formatted

**Comment by tboerger** [comment]
Dann, got to add back the bindata package

**Comment by tboerger** [comment]
Ready for review

**Comment by andreynering** [comment]
Trusted LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**